felix/bpf-gpl/tc.c

```diff
@@ -1687,6 +1687,7 @@ int calico_tc_skb_send_icmp_replies(struct __sk_buff *skb)
 	ctx->state->sport = ctx->state->dport = 0;
 	return forward_or_drop(ctx);
 deny:
+	(void)fib_flags;
```

Comment:
Suppress `unused-but-set-variable` warning since clang 13+.

```
$ clang-15 -Wall -Werror -fno-stack-protector -O2 -target bpf -emit-llvm -g -I ./include/libbpf/src/ -I ./include/libbpf/include/uapi -I/usr/include/x86_64-linux-gnu `./calculate-flags from_tnl_debug_co-re_v6.ll` -c tc.c -o from_tnl_debug_co-re_v6.ll
Flags: -DBPF_CORE_SUPPORTED -DCALI_LOG_LEVEL=CALI_LOG_LEVEL_DEBUG -DCALI_DROP_WORKLOAD_TO_HOST=false -DCALI_FIB_LOOKUP_ENABLED=false -DIPVER6 -DCALI_COMPILE_FLAGS=7
tc.c:1647:8: error: variable 'fib_flags' set but not used [-Werror,-Wunused-but-set-variable]
        __u32 fib_flags = 0;
              ^
tc.c:1647:8: error: variable 'fib_flags' set but not used [-Werror,-Wunused-but-set-variable]
        __u32 fib_flags = 0;
              ^
1 error generated.
```